src/parts/Portal/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
```

Comment:
I'd like to move away from this pattern to importing only what we need, i.e. `import { useEffect, useState }`. This is something we'll change as we go along (i.e. we're not  going to go back through existing features) but for new code we should start doing it now.

Reply:
Thanks for pointing this out! I prefer to not import full package too. I did it here like this just for the consistency since it was part of `parts` directory, but now I will write all new code with specific imports :)